.circleci/config.yml

```diff
@@ -187,6 +187,33 @@ jobs:
 
               CI_COMMIT_BRANCH="master" python3 ./Tests/scripts/update_conf_json.py
               CI_COMMIT_BRANCH="master" ./Tests/scripts/validate.sh
+  test-lint:
+      docker:
+        - image: devdemisto/content-build:3.0.0.12681
+      steps:
+        - checkout
+        - setup_remote_docker
+        - attach_workspace:
+            at: ~/project
+        - run:
+            name: Test lint on python and ps files
+            when: always
+            command: |
+              cd ~/project
+              export SDK_LINT_FILES_CHANGED=$(git diff master... --name-only -- demisto_sdk/commands/lint)
+
+              if [[ -n "${SDK_LINT_FILES_CHANGED}" ]]; then
+                echo "lint files changed running lint"
+                pip install . >> /dev/null
```

Comment:
you can just activate the tox and run the sdk from within it no?
see: `. .tox/py37/bin/activate` in validate-test section